.github/workflows/dashboard.yml

```diff
@@ -73,8 +73,8 @@ jobs:
                 issue_query: 'repo:aws/s2n-tls is:open is:pr review:approved -label:do_not_merge'
                 color: 'green'
               - type: 'number'
-                title: 'No review (external) last 14 days'
-                issue_query: 'repo:aws/s2n-tls is:open is:pr review:none -label:status/stale -label:s2n-core created:>{{ date("-14 days") }}'
+                title: 'No comment (external) last 14 days'
```

Comment:
Nit: I'd probably leave the title as "review" since comments fit our definition of review.